analytic_wip_report/README.rst

```diff
@@ -81,6 +81,7 @@ Contributors
 
 * Eficent <http://www.eficent.com>
 * Matmoz d.o.o. <http://www.matmoz.si>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```

Comment:
IMHO it should be:
* Jordi Ballester <jordi.ballester@eficent.com>
* Matjaž Mozetič <m.mozetic@matmoz.si>
* Aaron Henriquez <ahenriquez@eficent.com>
* Nikul Chaudhary<nikul.chaudhary.serpentcs@gmail.com>

but it's not big deal as it was like that before